sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_configure.py

```diff
@@ -10,7 +10,7 @@
 from opentelemetry._events import _set_event_logger_provider
 from opentelemetry._logs import set_logger_provider
 from opentelemetry.instrumentation.dependencies import (
-    get_dist_dependency_conflicts,
+    DependencyConflictError,
```

Comment:
Question: What's going to happen when someone doesn't have the latest version with the updated dependency resolution logic?

Reply:
Shound't opentelemetry-instrumentation version be locked? We're pulling in `opentelemetry-instrumentation 0.53b0` as a submodule of this it basically breaks all versions of `azure-monitor-opentelemetry`

Reply:
This is the error: 

```importError: cannot import name 'get_dist_dependency_conflicts' from 'opentelemetry.instrumentation.dependencies'```